src/components/Avatar/Avatar.tsx

```diff
@@ -0,0 +1,13 @@
+import React from "react";
+
+import "./Avatar.css";
+
+export type AvatarProps = {
+  firstChar?: string;
+};
+
+const Avatar: React.FC<AvatarProps> = ({ firstChar = "K" }) => (
+  <div className="avatar">{firstChar}</div>
```

Comment:
надо было сделать так

```suggestion
  <div className="avatar">{src ? (
        <img src={src} alt={alt || "avatar"} className="avatar__logo-img" />
      ) : (
        letter && <p className="avatar__logo-text"> {letter} </p>
      )}</div>
```